vignettes/decorate-modules-output.Rmd

```diff
@@ -0,0 +1,389 @@
+---
+title: "Customizing Module Output"
+author: "NEST CoreDev"
+output:
+  rmarkdown::html_vignette:
+    toc: true
+vignette: >
+  %\VignetteIndexEntry{Customizing Module Output}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+## Introduction
+
+`teal` is a powerful `shiny`-based framework with built-in modules for interactive data analysis. 
+This document outlines the customization options available for modifying the output of `teal` modules. 
+You will learn how to use `teal_transform_module` to modify and enhance the objects created by `teal` modules, 
```

Comment:
"modules" is a shiny term and it refers to any module. Decorate is dedicated to modify objects in `teal_module` - these created by `module()` function.

Reply:
Should I rephrase to 

> `teal` is a powerful `shiny`-based framework with built-id tools for interactive data analysis. 
This document outlines the customization options available for modifying the output of `teal` modules. 
You will learn how to use `teal_transform_module` to modify and enhance the objects created by `teal::module()`,

Reply:
60eec6e8fc66db73d4d4a56d4979d2dfee59580f